README.md

```diff
@@ -34,6 +34,8 @@ cloud_sql_proxy takes a few arguments to configure what instances to connect to
   IP. Defaults to `PUBLIC,PRIVATE`
 * `-term_timeout=30s`: How long to wait for connections to close before shutting
   down the proxy. Defaults to 0.
+* `--skip_failed_instance_config`: Setting this flag will allow you to prevent the proxy from terminating when
```

Comment:
While I prefer unix style flags, currently it's consistent with the rest of the binary to use one. 
```suggestion
* `-skip_failed_instance_config`: Setting this flag will allow you to prevent the proxy from terminating when
```

Reply:
You're right; done.